### PR TITLE
Use insights client to entitle AWS hosts to the customer portal

### PR DESCRIPTION
**Pull request by gravitypriest**

### CODE REVIEW

File: insights/client/apps/aws/__init__.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+from urllib3.exceptions import MaxRetryError
+from insights.client.connection import InsightsConnection
+from insights.client.schedule import get_scheduler
+from insights.client.constants import InsightsConstants as constants
+from insights.client.utilities import write_to_disk
+
+logger = logging.getLogger(__name__)
+net_logger = logging.getLogger('network')
+
+IDENTITY_URI = 'http://169.254.169.254/latest/dynamic/instance-identity'
```
Comment (subpop) [inline insights/client/apps/aws/__init__.py]: This isn't a publicly routeable IP, and I can't see where it gets updated to a routeable IP address. Is there some external factor that modifies this variable at runtime?

Comment (gravitypriest) [reply insights/client/apps/aws/__init__.py]: Yeah, check this: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html

Comment (subpop) [reply insights/client/apps/aws/__init__.py]: Ah ha, so it's an AWS thing. Perfect.

**Comment by patchkez** [comment]
test this please

**Comment by patchkez** [comment]
test this please

**Pull request merged**